### DISCUSSION TITLE
Setup Q: Do I need to install postgresql14 and redis docker apps if I use the compose (official) method?

### DISCUSSION THREAD

**Discussion by DMcWetty** [immich-app/immich#8843] [Q&A] [answered]
I used the official compose method to setup Immich on Unraid and everything is working really well. I went back and noticed that there were numerous docker apps added during setup. Two of them are immich_postgres and immich_redis apps. Do I need to have the "proper" Docker apps for those also installed or are they just redundant? Immich seems to work with or without them, but I didn't want to break any kind of functionality.
<img width="506" alt="Screenshot 2024-04-16 at 10 03 15 AM" src="https://github.com/immich-app/immich/assets/23362382/4c528eca-dc9c-42aa-9da7-c6d187428b45">

**Comment by schuhbacca** [comment, marked as answer]
You do not. If you used the docker compose method then postgres and redis are included. You don't need to install anything additional for immich. The compose method just "pins" specific versions of items. Pay attention to future release notes and make the compose updates if there are any.
